src/components/ArticleById.jsx

```diff
@@ -1,42 +1,66 @@
-import { getArticleById } from "../api";
+import { getArticleById, updateVote } from "../api";
+import ArticleComments from "./ArticleComments";
 
 import React, { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 
-import Navbar from "./Navbar";
 import { IconButton } from "@mui/material";
 import CommentTwoToneIcon from "@mui/icons-material/CommentTwoTone";
 import ThumbUpTwoToneIcon from "@mui/icons-material/ThumbUpTwoTone";
 
 const ArticleById = () => {
   const { article_id } = useParams();
-  const [article, setArticle] = useState([]);
+  const [article, setArticle] = useState({});
+  const [voteCounter, setVoteCounter] = useState();
+
+  const [disable, setDisable] = useState(false);
 
   useEffect(() => {
     getArticleById(article_id).then(res => {
       setArticle(res);
+      setVoteCounter(res.votes);
     });
   }, [article_id]);
 
+  console.log(article);
+  const handleVote = () => {
+    if (disable === false) {
+      updateVote(article_id, 1).catch(() =>
+        alert("Error can not update votes")
+      );
```

Comment:
I would much prefer to see an error message conditionally rendered rather than an alert. Sometime alerts won't show due to popups being disabled

Reply:
Done